src/index.ts

```diff
@@ -20,14 +21,15 @@ const renderUsers = async (config: ApiConfig) => {
 const app = () => {
   const config = (window as any).__CONFIG__;
   delete (window as any).__CONFIG__;
-
-  renderUsers(config.api);
+  createIoCContainer().register('config', config);
```

Comment:
you shouldn't create many containers. Please create a single container, then reuse it.

```
const ioc = createIoCContainer();

...
ioc.resolve('users');
...
ioc.register('config', config);
```